workspace-manager-clienttests/README.md

```diff
@@ -61,6 +61,35 @@ export TEST_RUNNER_SERVER_SPECIFICATION_FILE="workspace-dev.json"
 Each service account JSON files in the resources/serviceaccounts directory of this project specifies a default file
 path for the client secret file. This default path should match where the render-config.sh script puts the secret.
 
+#### Test runner SA keys
+The test runner SA key is necessary for uploading test results to Google Buckets and BigQuery. 
+Each environment has its own test runner SA key with the right permissions for uploading test results.
+The provisioning of the test runner SA keys will be done with `Terraform` in collaboration with `devOps`.
+Additional `k8s` permission will allow test runner to manipulate a cluster for resiliency tests.
+
+Here's the vault paths of various test runner SA keys.
+
+Static environments (alpha, dev, perf, qa, staging)
+One test runner SA key for each static environment.
+`secret/dsde/terra/kernel/<env>/common/testrunner-sa`
+
+`<env> = { alpha, dev, perf, qa, staging }`
+
+Preview environments
+One common test runner SA key for all preview environments.
+`secret/dsde/terra/kernel/integration/preview/testrunner-sa`
```

Comment:
The test runner SA key for this PR was created manually. Though the intention is for `devOps` to review this PR also to see if falls within devOps best practices of provisioning these keys with `Terraform`. Yes `k8s` admin permissions is also part of `devOps` review.

Reply:
Got it, thanks for talking with DevOps about it. I think we should not include this in the README until DevOps has signed off and we have made the Terraform changes. Otherwise, it is misleading.

Reply:
Yes, I'll share the steps (during today's `devOps` syncup) on provisioning the Test Runner SA account with the permissions we need.